### DISCUSSION TITLE
docker+onlyoffice+aria2/qbittorrent+AriaNg+AList集成

### DISCUSSION THREAD

**Discussion by homuraLan** [AlistGo/alist#4565] [Show and tell]
https://github.com/Sincejunly/officeaanglist.git

**Comment by homuraLan** [comment]
集成预览效果，简单配置就可以实现，多人用户协作，访客可以看不能改，

**Comment by homuraLan** [comment]
![image](https://github.com/alist-org/alist/assets/96775034/30eb1b28-bd80-41ca-965b-9fff6e37cfe3)

**Comment by BenzLeung** [comment]
@Sincejunly 大佬你是把 onlyoffice 和 alist 和 aria 揉在一起合成一个镜像了吗？这样是不是 以后不好更新？

**Comment by homuraLan** [comment]
我以后来你更新就好，我会放出版本信息

BenzLeung ***@***.***> 于2023年6月29日周四 14:11写道：

**Comment by BenzLeung** [comment]
@Sincejunly 建议弄成分离的吧，让用户在 docker-compose 中填入alist和onlyoffice的地址和密钥，并让用户自行配置alist的预览参数，你的docker镜像只负责对接、中转、edit.html。这样的话你这镜像不需要频繁更新，也不用总是盯着alist和onlyoffice的版本同步了。

**Comment by homuraLan** [comment]
![image](https://github.com/alist-org/alist/assets/96775034/b11ce507-19cc-4690-bb16-52f5acd89747)  
PDF 更新支持可以转换图片dpf成文字pdf
